bucket/rednotebook.json

```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "https://rednotebook.sourceforge.io/",
+    "description": "A modern journal",
```

Comment:
```suggestion
    "description": "Graphical diary and journal",
```

Reply:
not many graphical functions are available but you can see images and use latex, but why not